flowmachine/tests/server/test_server_config.py

```diff
@@ -14,11 +14,15 @@ def test_get_server_config(monkeypatch):
     monkeypatch.setenv("FLOWMACHINE_PORT", 5678)
     monkeypatch.setenv("FLOWMACHINE_SERVER_DEBUG_MODE", "true")
     monkeypatch.setenv("FLOWMACHINE_SERVER_DISABLE_DEPENDENCY_CACHING", "true")
+    monkeypatch.setenv("FLOWMACHINE_CACHE_PRUNING_FREQUENCY", 1)
+    monkeypatch.setenv("FLOWMACHINE_CACHE_PRUNING_TIMEOUT", 2)
     config = get_server_config()
-    assert len(config) == 3
+    assert len(config) == 5
     assert config.port == 5678
-    assert config.debug_mode == True
-    assert config.store_dependencies == False
+    assert config.debug_mode
```

Comment:
Hmm. This caught me out in the past - `config.debug_mode` was getting set to `"true"` instead of `True`. Being a truthy string, `config.debug_mode="true"` would still pass this test.